wyvern/components/features/feature_store.py

```diff
@@ -69,7 +73,9 @@ async def fetch_features_from_feature_store(
         )
 
         if response.status_code != 200:
-            logger.error(f"Error fetching features from feature store: {response}")
+            logger.error(
+                f"Error fetching features from feature store: [{response.status_code}] {response.json()}",
```

Comment:
This log didn't have enough info before  because it was calling `response.__repr__`
```
    def __repr__(self):
        return f"<Response [{self.status_code}]>"
```

This way, the log looks like:
```
ERROR: [2023-08-17 00:10:05,385] [wyvern.components.features.feature_store] Error fetching features from feature store: [403] {'detail': 'Invalid credentials'}
```